### PR TITLE
Update canary

### PR DESCRIPTION
**Pull request by loweisz**
# Purpose of PR

## PR Checklist

- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Typescript typings are added/updated/not required

### CODE REVIEW

**Review by shikaan — APPROVED** [review]

**Review by andipaetzold — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -150,6 +150,6 @@
     ]
   },
   "dependencies": {
-    "contentful-management": "^7.23.0"
+    "contentful-management": "7.27.0"
```
Comment (andipaetzold) [inline package.json:153]: Why did you pin the dependency?

```suggestion
    "contentful-management": "^7.27.0"
```

**Review by andipaetzold — CHANGES_REQUESTED** [review]
Why did you pin the dependency?

File: package.json (hunk 1 above)
Comment (loweisz) [reply package.json:153]: it was pinned in master when it was under `devDependencies`: 
https://github.com/contentful/ui-extensions-sdk/blob/1b474e136ff75cda86f20036e00a31f51bef29da/package.json#L58

Comment (andipaetzold) [reply package.json:153]: It's fine to pin dev dependencies as people installing app-sdk do not install those. But regular dependencies should not be pinned as we fix the cma.js our users can use which will cause issues.

Comment (loweisz) [reply package.json]: Makes sense 👍

**Review by andipaetzold — APPROVED** [review]

**Pull request merged**